Contributions/Using_Linux_in_a_Safe_System.md

```diff
@@ -0,0 +1,348 @@
+
+[License: CC BY-SA 4.0](#License-CC-BY-SA-40)
+
+## **Terms and Abbreviations**
+
+   See the matching section in *Interference Scenarios for an ARM64 Linux System*
+
+## **References**
+
+1. ***Interference Scenarios for an ARM64 Linux System***
+2. ***Safety Requirements for a Generic Linux System***
+3. ***CheckList for Safety Claims on a Generic Linux System***
+4. [CC BY-SA 4.0 Deed | Attribution-ShareAlike 4.0 International | Creative Commons](https://creativecommons.org/licenses/by-sa/4.0/) - <https://creativecommons.org/licenses/by-sa/4.0/> License
+
+
+## **Purpose of the document**
+The considerations provided here are meant to help with designing a safe
+system incorporating Linux.
+
+There are many reasons why one might want to use Linux in a system, but its
+safety is certainly not one of them - nor should it be.
+
+Linux is neither conceived, nor evolved, to be safe. To be fair, it has
+never made any claim to be safe, therefore it should not be expected to be safe.
+Safety is not something that happens by accident. These considerations are
+intended to help with designing a system that is safe **despite** using Linux.
+
+All of the background justification, related to hardware (ARM64), Linux kernel
+architecture, and allocation of safety requirements, can be found in the documents
+mentioned in the [References](#References) section.
+
+In particular, the present document can be seen as a counterpart to the document
+**CheckList for Safety Claims on a Generic Linux System**.
+While the Checklist is meant to help with detecting problems, the present document
+wants to help with avoiding problems, upfront.
+
+The same disclaimer made for the checklist is valid here too: this document is
+not meant to provide full solutions to the problems listed.
+Solutions are often heavily contextualised, and must be designed to suit specific
+needs, in terms of use cases, safety requirements and system design.
+
+The evaluation of the very same mitigation strategy can range from overkill to
+insufficient, depending on where/how it is applied.
+
+
+## **Structure of the document**
+- The document starts with a section containing design considerations.
+- The following section contains several examples of how to apply such
+  considerations to the design of the Linux-bearing system.
+
+
+## **Considerations**
+
+**Premise:** some of the following considerations assume a fairly high level
+of freedom in designing the overall system.
+While one must acknowledge that in practice this freedom will not be always
+available, due to design constraints, it is equally true that attempting to
+cope with these constraints can lead to an even higher total cost of ownership.
+
+* **Clearly define requirements toward system integrity and availability, upfront**
+
+  Having to simply report failures (within a maximum allotted time - and how much?)
+  usually is significantly simpler than having to guarantee that a certain
```

Comment:
As per my experience a system don't have to guarantee that a certain safety related functionality will not be interrupted. 
If it is not possible to ship a safety related functionality system should move to a different state applying recovery actions.